.github/workflows/maven.yml

```diff
@@ -0,0 +1,74 @@
+name: Arquillian-Cube Maven Build
```

Comment:
Please drop the dash, the project name is "Arquillian Cube" (no dash).

Never the less I always struggle how to name these. The project name seems redundant. The maven build is also redundant, since that's the only build system we use here. Also, build is a bit of a misnomer as primarily this is used to testing...